### ISSUE TITLE
Implement Authentication aspect (in a form of context manager)

### ISSUE THREAD

**Issue by s-kostyuk** [s-kostyuk/everpl#65]
Implement an Aspect that will control Authorization in the system. I.e. will check if the current user is able to (is authorized to) execute a requested action.

Use Python's Context Managers and a concept of Ambient Context for passing of authentication tokens, use Interceptor pattern as an implementation detail for controlling of calls to Services.

Blocked by #63

**Comment by s-kostyuk** [comment]
Postponed. Current issue: how to pass a requested action to authorization checker
